cmd/abapEnvironmentPullGitRepo.go

```diff
@@ -200,13 +206,62 @@ func getHTTPResponse(requestType string, connectionDetails connectionDetailsHTTP
 	return req, err
```

Comment:
Here, you should also check `if err != nil` and return with `return req, fmt.Errorf()` if error occurs